src/commands/cluster-addslots.json

```diff
@@ -7,6 +7,7 @@
         "arity": -3,
         "container": "CLUSTER",
         "function": "clusterCommand",
+        "feature_flags": ["legacy-cluster"],
```

Comment:
i agree it's a good idea, but let's take it in a follow up PR.
the purpose of this one is a for a quick shifting of code from one place to the other, with minimal or actually no change of behavior.
we need it to be merged quickly so that we don't accumulate conflicts, and we need it to be done carefully so that we keep as much history as possible.
logical changes can be applied later, just let's try to avoid renaming or moving the same thing twice.

Reply:
If it's just a quick shift, let's remove all the code generation stuff and leave the commands as is. None of this is strictly needed for the shifting. We can restructure the commands when we have the extensions sorted.

Reply:
Yes, that would be easiest!